components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -455,24 +456,19 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
         ClaimMapping[] coreClaims;
         ClaimMapping[] extensionClaims = null;
 
-        User scimUser;
         int totalUserCount;
         try {
             String[] userNames = null;
-            if (!SCIMConstants.UserSchemaConstants.GROUP_URI.equals(attributeName)) {
 
-                if (!filterOperation.equalsIgnoreCase(SCIMCommonConstants.EQ)) {
-                    String error = "Filter operator "+ filterOperation +" is not implemented non group filtering";
-                    throw new NotImplementedException(error);
-                }
+            if (isNotGroupFilteringSupported(filterOperation)) {
```

Comment:
shouldn't this be isNotUserFilteringSupported ?

Reply:
I use this function in group filtering as well. Therefore changed it to isNotFilteringSupported.